### PR TITLE
Update Token List with correct IDs

### LINKED ISSUE #88: Incorrect Token IDs from tokenList.json
The following two tokens have incorrect IDs:

https://github.com/gnosis/dex-js/blob/0d93c1c36f0323b693add5a4697b2cbd585e1b23/src/tokenList.json#L120-L143

Token 11: is WBTC
Token 12: is SAI

### PR DESCRIPTION
**Pull request by bh2smith**
Check, via `tokenIdToAddressMap` on etherscan - https://etherscan.io/address/0x6F400810b62df8E13fded51bE75fF5393eaa841F#readContract that the following tokens have the ids as updated here.

Closes #88

### CODE REVIEW

**Review by anxolin — APPROVED** [review]
Great catch!

The issue is that we are not getting the id from the smart contract. I believe we need to address this asap to prevent this issues.

This was flagged by Leandro already

File: src/tokenList.json [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+    "symbol": "OWL",
+    "decimals": 18,
+    "addressByNetwork": {
+      "1": "0x1A5F9352Af8aF974bFC03399e3767DF6370d82e4",
+      "4": "0xa7D1C04fAF998F9161fC9F800a99A809b84cfc9D"
+    },
+    "website": "https://gnosis.io/tokens",
+    "description": "OWL can be used to pay 1 USD in fees on Gnosis platforms",
+    "rinkeby_faucet": null
+  }
```
Comment (alfetopito) [inline src/tokenList.json]: Missing a comma

**Review by alfetopito — CHANGES_REQUESTED** [review]
Tiny issue with JSON formating

**Comment by bh2smith** [comment]
My only concern now is that this change will result in OWL being listed first in the dropdown menu on the interface.

**Comment by alfetopito** [comment]
> My only concern now is that this change will result in OWL being listed first in the dropdown menu on the interface.

You can just move it down the list and keep the id `0`.
AFAIK we don't sort based on id and use instead the array position.

**Review by alfetopito — APPROVED** [review]

**Pull request merged**